api/organisation/index.ts

```diff
@@ -21,7 +21,24 @@ export async function handleOrganisationRoute(req: Request, res: Response) {
 }
 
 export function getOrganisationDetails(req: Request, url: string): Promise<AxiosResponse> {
-  return req.http.get(`${url}/refdata/external/v1/organisations`);
+  return req.http.get(`${url}/refdata/external/v2/organisations`);
+}
+
+// delete next two functions after register org feature not necessary
+export async function handleOrganisationV1Route(req: Request, res: Response) {
+  try {
+    const response = await req.http.get(
+      `${getConfigValue(SERVICES_RD_PROFESSIONAL_API_PATH)}/refdata/external/v1/organisations`
+    );
+    res.send(response.data);
+  } catch (error) {
+    const errReport = {
+      apiError: error.data.message,
```

Comment:
I think its better to use next [global error handler] instead of handling the error here.

Reply:
not actually my code, just copied and pasted, but I will try to resolve this anyway